internal/fs/inode/file.go

```diff
@@ -603,5 +603,6 @@ func convertObjToMinObject(o *gcs.Object) (mo storage.MinObject) {
 		MetaGeneration: o.MetaGeneration,
 		Updated:        o.Updated,
 		Metadata:       o.Metadata,
+		IsGzipEncoded:  o.ContentEncoding == "gzip",
```

Comment:
Use directly contentEncoding property instead of deriving isGzipEncoded. It will provide extensibility incase we need to handle more content encoding types.

Reply:
Ok. Changed accordingly.